src/mobile/android/gradle.properties

```diff
@@ -10,7 +10,6 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx1536m
```

Comment:
This is necessary in order to allocate enough memory for the build process. The deployment on CI may fail if this is removed